Header  header
bool    flag
uint32  ID_ROBOT
uint32  capacity